modules/about-monitoring-stack-components.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
```

Comment:
This file isn't included in the assembly.

Reply:
The file has been re-named .  Content is the same. Based on previous feedback.